src/main/java/redis/clients/jedis/resps/LCSMatchResult.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2011-2021 the original author or authors.
```

Comment:
@dengliming As you are the original author of this (or similar) class, could you please give us your permission to remove the license from here?